### PR TITLE
Use our own asset pipeline on top of sprockets 3

### PR DESCRIPTION
**Pull request by romaingweb**
This is an attempt to get rid of sinatra-asset-pipeline.  This library depends on an old version of sprockets which depends on an old version of tilt.  We use tilt heavily to render the website so it would be nice to be on the latest version.

Inspiration: http://javivelasco.com/blog/badass-asset-pipeline

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -26,6 +30,16 @@ task :backup_events do
...
 end
 
-require 'sinatra/asset_pipeline/task'
-require './apps/static/app'
-Sinatra::AssetPipeline::Task.define! Cucumber::Website::Static::App
+namespace :assets do
+  desc 'Precompile assets'
+  task :precompile do
+    environment = Cucumber::Website::Static::App.assets
+    manifest = Sprockets::Manifest.new(environment.index, File.join(Cucumber::Website::Static::App.assets_path, "manifesto.json"))
```
Comment (mattwynne) [inline Rakefile]: manifest_o_? Is that a typo or a joke?

File: lib/cucumber/website/asset_pipeline.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
+require 'sprockets'
+require 'sprockets-helpers'
+require 'autoprefixer-rails'
+
+module Cucumber
+module Website
```
Comment (mattwynne) [inline lib/cucumber/website/asset_pipeline.rb:6]: Do you think this should go into a `Static` namespace?

Comment (romaingweb) [inline lib/cucumber/website/asset_pipeline.rb:6]: I'm not sure. This pipeline could be reusable. But in this case, I guess we could put in under the static namespace inside the static app even.

Comment (mattwynne) [inline lib/cucumber/website/asset_pipeline.rb:6]: Yeah. I like having it in a separate module I think. I'm a bit unclear what code we should put in `apps` and what we put in `lib`. I'm inclined to say we should keep most of it in `lib` but be better about namespacing it so it's clear what code belongs to what app, and what (if any) is shared / reusable.

**Comment by mattwynne** [comment]
connected to #129

**Pull request merged**